### DISCUSSION TITLE
Get Hl7.Fhir.Model as a standalone package?

### DISCUSSION THREAD

**Discussion by ghost** [FirelyTeam/firely-net-sdk#2213] [Ideas]
I have a multi-part application and all of them need to create/serialize/de-serialize/use FHIR data. Is there a way to get just the data type classes (mainly things under the `Hl7.Fhir.Model` namespace) as a standalone package or library? That way, I don't need to provide other parts of the application features/extensibility points they should not be having.

If it matters platform is `.NET 6.0`.

**Comment by ghost** [comment]
Ping?

**Reply by marcovisserFurore** [reply]
Hi @sujayvsarma, the datatypes and resources are in the `Hl7.Fhir.R4` package. Also in this package is the FhirClient. We are planning for version 5 to rearrange the libraries. The idea is that datatypes and resources would come in a separate library. This is a major change and involves a lot of work, but it is on our roadmap.

**Reply by ghost** [reply]
Thank you @marcovisserFurore !
